Sources/SourceKitLSP/Swift/MacroExpansion.swift

```diff
@@ -74,51 +74,17 @@ extension SwiftLanguageService {
     var macroExpansionFilePaths: [URL] = []
     for macroEdit in expansion.edits {
       if let bufferName = macroEdit.bufferName {
```

Comment:
It’s a shame that we need to effectively run the expand macro refactoring request twice now: Once to get the macro’s range and buffer name and then again in the `workspace/getReferenceDocument` request. But I don’t really see a way around that. So, I guess it’s fine.